Modules/Loadable/VolumeRendering/MRMLDM/vtkMRMLVolumeRenderingDisplayableManager.cxx

```diff
@@ -1079,7 +1079,7 @@ void vtkMRMLVolumeRenderingDisplayableManager::vtkInternal::FindPickedDisplayNod
 
 //---------------------------------------------------------------------------
 vtkMRMLVolumeRenderingDisplayableManager::vtkMRMLVolumeRenderingDisplayableManager()
-: VolumeRenderingLogic(nullptr)
+
```

Comment:
Would it be possible to remove the additional empty line like this one ?

Reply:
@jcfr. An autoformatter is really needed.  We should employ clang-format or similar if that level of formatting consistency is desired.